pages.pt_BR/common/fd.md

```diff
@@ -0,0 +1,29 @@
+# fd
+
+> Uma alternativa para `find`.
+> Visa ser mais rápido e fácil de usar do que `find`.
+> Mais informações: <https://github.com/sharkdp/fd>.
+
+- Encontra recursivamente arquivos que correspondam ao padrão fornecido no diretório atual:
+
+`fd {{padrão}}`
+
+- Encontra arquivos que começam com "foo":
```

Comment:
The following description fix could be applied:

```suggestion
- Encontra arquivos que começam com `foo`:
```

Our style guide doesn't specify whether this fix should be applied or not.